python/dgllife/model/gnn/pagtn.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Path-Augmented Graph Transformer Network
+# pylint: disable= no-member, arguments-differ, invalid-name
+
+import torch
+import torch.nn as nn
+from dgl.nn.functional import edge_softmax
+import dgl.function as fn
+
+__all__ = ['PAGTNGNN']
+
+class PAGTNLayer(nn.Module):
+    """
+    Single PAGTN layer from `Path-Augmented Graph Transformer Network
+    <https://arxiv.org/abs/1905.12712>`__
+
+    This will be used for incorporating the information of edge features
+    into node features for message passing.
+
+    Parameters
+    ----------
+    node_in_feats : int
+        Size for the input node features.
+    node_out_feats : int
+        Size for the input edge features.
+    edge_feat_size : int
```

Comment:
For consistency, it might be better to use "edge_feats" rather than "edge_feat_size".